docs/www/quick-start-kubernetes.md

```diff
@@ -52,13 +52,23 @@ You can either create a Kubernetes cluster on your local machine or on a cloud p
 
   <tab id="Kind">
 
-  [Kind](https://kind.sigs.k8s.io) is a tool that lets you create local Kubernetes clusters using Docker.
-    After you install Kind, set up a cluster with:
+  [Kind](https://kind.sigs.k8s.io) is a tool that lets you create local Kubernetes clusters using Docker. 
+    After you install Kind, verify that port 30001 is free on your machine before you set up a cluster or cluster creation will fail. Set up a cluster with:
 
   ```
-  kind create cluster
+  kind create cluster --config docs/kind-external.yaml
   ```
 
+  ```
+kind: Cluster
+apiVersion: kind.x-k8s.io/v1alpha4
+nodes:
+- role: control-plane
+  extraPortMappings:
+  - containerPort: 30001
+    hostPort: 30001
+```
```

Comment:
I don't understand why, but when I click on the copy button on the right upper corner the content is miss aligned

```yaml
kind: Cluster
apiVersion: kind.x-k8s.io/v1alpha4
nodes:
- role: control-plane
extraPortMappings:
- containerPort: 30001
  hostPort: 30001
``` 

Everything bellow extraPortMappings is miss align. It misses two spaces
![Screenshot 2022-01-18 at 09 54 52](https://user-images.githubusercontent.com/22834495/149903496-996c56fe-0a7b-46f6-ae61-e11d6be00c91.png)

Reply:
This is probably the problem of preview not being refreshed.